charts/fluentd/fluentd-elasticsearch/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: fluentd
+description: A Helm chart for Kubernetes
```

Comment:
Should the description and/or name indicate that this is specific to elasticsearch ?